tests/unit/bokeh/test_settings.py

```diff
@@ -158,6 +159,10 @@ def test_convert_logging_bad(self) -> None:
         with pytest.raises(ValueError):
             bs.convert_logging("junk")
 
+    @pytest.mark.parametrize("value", ["none", "errors", "all"])
+    def test_convert_validation_good(self, value) -> None:
+        assert bs.convert_validation(value) == value
```

Comment:
I missed that there was not a `test_convert_validation_bad` added. e.g. something like
```
    def test_convert_validation_bad(self) -> None:
        with pytest.raises(ValueError):
            bs.convert_validation("junk")
```
It's always important to probe the space of inputs to include bad values, to make sure that the expected behavior in those cases is intended, documented, and repeatable.

Reply:
Thanks for catching that! The test for that has been added to test_settings.py now.